dynaboard.yml

```diff
@@ -66,25 +72,41 @@ nodes:
         - nodeID: 48d9fd9c-86a9-421f-9c3a-b7d595524ed1
           orderKey: ZzV
     properties:
+      footerProps:
+        element: ''
+      headerProps:
+        element: ''
+      includeInSitemap: '{{true}}'
       layoutStyle: VERTICAL_STACK
-      minWidth: 768px
```

Comment:
I'm not sure why the min-width changed? May have been me, but it should be tablet sized.

Reply:
I think the default changed, so its being dropped from export